src/zeusops_bot/reforger_config_gen.py

```diff
@@ -23,12 +37,13 @@ def __init__(self, base_config_file: Path, target_folder: Path):
         """Instantiate a config generator with the base config to use"""
         self.base_config = base_config_file
         self.target_dest = target_folder
+        instance.config = self
 
     def zeus_upload(
         self,
         scenario_id: str,
         filename: str,
-        modlist: list[ModDetail] | None,
+        modlist: str | None,
```

Comment:
Disagree, as we said elsewhere: this is the core business logic, we NEED the modlist to be already mostly parsed, and something calling this func should be the one doing the parsing (in cli.py or discord.py depending).

Strongly recommend revert to pass `list[ModDetail]`.

Reply:
Huh, this was supposed to be fixed in the other PR already, must've messed up a rebase or something. In any case, this is just an accidental leftover

Reply:
Rebased on top of the other PR, this is fixed